packages/mobile/src/components/premium-track-purchase-drawer/AudioMatchSection.tsx

```diff
@@ -0,0 +1,42 @@
+import { View } from 'react-native'
+
+import { makeStyles } from 'app/styles'
+
+import { Text } from '../core/Text'
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  root: {
+    backgroundColor: palette.secondary,
+    border: palette.secondaryDark2,
+    paddingHorizontal: spacing(8),
+    paddingVertical: spacing(2)
+  },
+  text: {
+    textAlign: 'center'
+  }
+}))
+
+const messages = {
+  earn: (amount: number) => `Earn ${amount} $AUDIO when you buy this track!`
+}
+
+type AudioMatchSectionProps = {
+  amount: number
+}
+
+export const AudioMatchSection = ({ amount }: AudioMatchSectionProps) => {
+  const styles = useStyles()
+  return (
+    <View style={styles.root}>
+      <Text
+        style={styles.text}
```

Comment:
I think it might be nice if `Text` could do the centering for you via a prop, instead of having to apply a one-off style each time.
cc @dylanjeffers thoughts?

Reply:
oh yeah.. just left this comment too. Agreed!